### PR TITLE
feat: added homepage for the e-com

### PR DESCRIPTION
**Pull request by xii-vi**
## Added e-Com Homepage
Home-RE-KICKS

### CODE REVIEW

File: stylesheet/homepage.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  background-color: transparent;
+  color: white;
+  font-size: 1.2rem;
+  top: 2rem;
+  letter-spacing: 0.5rem;
+  font-family: monospace;
+  text-align: center;
+}
+.card-wrapper:hover{
+  box-shadow: 3px solid black;
```
Comment (arjunbharti) [inline stylesheet/homepage.css]: You can use color variable here

**Review by arjunbharti — APPROVED** [review]
Just added a minor suggestion. Everything else is good. Nice work!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+                            <p class="bold-text">Air Force 1 GYPSY ROSE</p>
+                            <small>₹ 9,995</small>
+                        </div>
+                        <div class="card-badge p-2">GRAILS</div>
+                    </div>
+                </div>
+            </a>
+
+            <a href="/pages/sneaker.html">
+                <div class="card-wrapper">
```
Comment (devdeadviz) [inline index.html]: Try to use relative paths.

File: js/slideshow.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+let slideIndex = 0;
+showSlides();
+
+function showSlides() {
+    const slides = document.getElementsByClassName("slideshow-slides");
```
Comment (devdeadviz) [inline js/slideshow.js:5]: Use the same naming convention.

File: stylesheet/layout.css [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        display: grid;
+        grid-template-columns: 1fr;
+    }
+
+    .footer-section-layout{
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+    }
+
+
```
Comment (devdeadviz) [inline stylesheet/layout.css]: Use variables for breakpoints values in a media query.

**Review by devdeadviz — CHANGES_REQUESTED** [review]
Use the relative path method.

File: stylesheet/homepage.css (hunk 1 above)
Comment (xii-vi) [reply stylesheet/homepage.css]: _Suggested changes are made_

File: index.html (hunk 2 above)
Comment (xii-vi) [reply index.html]: Suggested changes are made

File: stylesheet/layout.css (hunk 4 above)
Comment (xii-vi) [reply stylesheet/layout.css]: media queries do not work with CSS native variables.

Comment (devdeadviz) [reply stylesheet/layout.css]: Okay

**Review by devdeadviz — APPROVED** [review]
Looks good to me!

**Pull request merged**